src/chapters/data-manipulation/data-manipulation.rst

```diff
@@ -1,2 +1,162 @@
 Data Manipulation
-=================
\ No newline at end of file
+=================
+
+.. _readingDataManipulation:
+
+Readings
+--------
+
+The readings and exercises are a little different for this chapter.  Both have been combined into 2 
+jupyter notebooks and the link can be found :ref:`here<dataManipulationNotebook>`. 
+
+| Complete the notebooks in the following order using the readings to help you:
+
+#. DataManipulationWorkbook
```

Comment:
Could we link to each notebook here?

Reply:
Because we are having them fork and clone the GitHub repo to their account, I link adding a link to the notebook contained in the repo might be confusing.  They are going to need to open the one they just cloned to their computer.

Reply:
Reworked both data-manipulation.rst and exercises.rst, Moved all the instructions to a note in the exercises page.  Add added two admonitions to the data-manipulation.rst file